### ISSUE TITLE
infinityfree.com: ads / breakage / detection

### ISSUE THREAD

**Issue by abigailitor** [uBlockOrigin/uAssets#25178]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://www.infinityfree.com/
```

### Description

When I enable uBO, websites appear broken. I’ve tried resetting it and even removed and re-added it, but the issue still persists.
- I am not using any other extensions, VPN, or DNS services.
- When I disable uBO, it works fine.

### Other extensions used

I am not using any other ad blockers.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/9485df8b-3673-45a1-a24e-43eabbc4959f">

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/0566a766-99e6-430e-84ee-6d9e9b8ba563">

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.59.0
Chromium: 128
filterset (summary):
 network: 152523
 cosmetic: 48110
 scriptlet: 22222
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 39789-122, now
  ublock-badware: 10915-6, now
  ublock-privacy: 1118-2, now
  ublock-unbreak: 2471-1, now
  ublock-quick-fixes: 215-73, now
  easylist: 85194-570, now
  easyprivacy: 53100-787, now
  urlhaus-1: 28220-0, now
  plowe-0: 3547-0, now
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 226 ms (selfie)
 maxAssetCacheWait: 19 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 16
 network:
  cloudflareinsights.com: 1
  fuseplatform.net: 1
  google.com: 7
  googlesyndication.com: 1
  googletagmanager.com: 1
  html-load.com: 2
  youtube-nocookie.com: 3
 extended:
  ##ins.adsbygoogle[data-ad-client]
  ##ins.adsbygoogle[data-ad-slot]
  ##+js(set-local-storage-item, adshield-analytics-uuid, $remove$)
  ##+js(set-local-storage-item, /_fa_dXVpZA==$/, $remove$)
  ##+js(set-local-storage-item, /_fa_Y2FjaGVfaXNfYmxvY2tpbmdfYWNjZ…
  ##+js(set-local-storage-item, /_fa_Y2FjaGVfaXNfYmxvY2tpbmdfYWRz$…
  ##+js(set-local-storage-item, /_fa_Y2FjaGVfYWRibG9ja19jaXJjdW12Z…
  ##+js(trusted-replace-argument, HTMLScriptElement.prototype.setA…
  ##+js(set-constant, ytInitialPlayerResponse.playerAds, undefined…
  ##+js(set-constant, ytInitialPlayerResponse.adPlacements, undefi…
  ##+js(set-constant, ytInitialPlayerResponse.adSlots, undefined)
  ##+js(set-constant, playerResponse.adPlacements, undefined)
  ##+js(json-prune, playerResponse.adPlacements playerResponse.pla…
  ##+js(json-prune-fetch-response, reelWatchSequenceResponse.entri…

```
</details>

**Comment by Rivaraav** [comment]
I'm having the same problem. How can I resolve this???

<details>
<summary>screenshot</summary>
<h3>Enable</h3>

![image](https://github.com/user-attachments/assets/bde6e0ea-54db-4597-b264-d946a4c69885)

<h3>Disable</h3>

![image](https://github.com/user-attachments/assets/79e74477-88bc-427b-b035-81862aa9b913)

</details>

**Comment by SolaraCelestial** [comment]
Oh, i posted about the same issue...
- https://github.com/uBlockOrigin/uAssets/issues/25179

**Comment by stephenhawk8054** [comment]
cc @JobcenterTycoon Looks like regex doesn't match with `replace`

**Comment by JobcenterTycoon** [comment]
Yes the code changed but the old filter works fine for now. Sadly `https://regex101.com/` is down at the moment.

**Comment by stephenhawk8054** [comment]
Hmm... When I disable all scriptlet filters `infinityfree.com#@#+js()`, I don't see `replace` being applied

![image](https://github.com/user-attachments/assets/93d7148d-2b4c-40eb-a263-fda89df98b73)

**Comment by stephenhawk8054** [comment]
Oh sorry, didn't know you have updated. Confirm the update works.

**Comment by JobcenterTycoon** [comment]
Works fine for me. Try to clear the website cache and click here to get the latest filterlist update https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-quick-fixes,ublock-filters&manual=1

**Comment by stephenhawk8054** [comment]
https://github.com/uBlockOrigin/uAssets/issues/26098

**Comment by sophia7209** [comment]
When uBO is enabled, the webpage breaks and requests to disable the adblocker, which suggests that uBO is being detected again.

**Comment by liruno** [comment]
Hi,
It seems that the CSS request for InfinityFree is being blocked.
You can reproduce the issue by enabling uBlock filters and refreshing the page, and it will immediately show the screen like the one below. Please check.
If need any additional information, please let me know.

![image](https://github.com/user-attachments/assets/03ff9181-5a42-4590-b9e2-8b332d14480b)

**Comment by liruno** [comment]
When I clicked the OK button, it took me to this page.

![image](https://github.com/user-attachments/assets/e9702302-8db9-4861-888e-2da1ba29b9a5)

**Comment by JobcenterTycoon** [comment]
We updating every day already but the results may be unstable. To get more stable results i recommend using the **tinyShield** userscript developed by korean filterlist maintainer. Link to the userscript:

--> [tinyShield Userscript](https://github.com/List-KR/tinyShield) <--

**Comment by Grendel7** [comment]
I was pointed here as this was considered a duplicate of an issue I just submitted.

I would like to report an issue with the same site. Specifically, some of the workaround code is breaking the CAPTCHAs, preventing people from logging in or signing up.

I'm not interested in a workaround for myself, I'm interested in a fix that doesn't result in all uBO users being unable to access my website.

**Comment by JobcenterTycoon** [comment]
Works all fine for me.

**Comment by Grendel7** [comment]
> Works all fine for me. Try clearing the website cache.

It seems that Stephen fixed the issue yesterday with this commit: https://github.com/uBlockOrigin/uAssets/commit/85479661b2b9c35ce0d7130696ef9e47ad83490e

I'm not quite sure why it works, but it appears to be working.

**Comment by Insoulum** [comment]
Re-check needed for https://dash.infinityfree.com/ (anti-adblock is back?)

> [!NOTE]
> <img width="776" height="372" alt="Image" src="https://github.com/user-attachments/assets/2eb07915-a8da-4dce-ae87-a5aecff48342" />

**Comment by JobcenterTycoon** [comment]
fixed now.

**Comment by Marko-98** [comment]
Updated all the filters and I still get this.

**Comment by Insoulum** [comment]
> Updated all the filters and I still get this.

Me too. I am sure that filter updates are released once a day. We need to wait, I suppose.

**Comment by JobcenterTycoon** [comment]
yes it need a few hours
